README.md

```diff
@@ -223,7 +219,16 @@ The server will respond with JSON object.
 - The applicationId returned can be used to view the candidate profile in Lever at the url: `https://hire.lever.co/search/application/{applicationId}`. Note that only users logged in to Lever will be able to access that page.
 - On error, we'll send the appropriate HTTP error code and a body of `{ok:false, error:<error string>}`.
 
+### WARNING: POST Application Rate limit
+
+**Application create requests are rate limited. Your team will need to properly handle `429` responses if you build a custom job application page.**
+
+To prevent spam and protect system integrity, Lever will return a `429` status code (`TOO MANY REQUESTS`) if your custom job site issues more than 2 application POST requests per second. Mitigate this by:
 
+* redirecting to the respective Lever-hosted application form
+* queueing the application requests
+* retrying the application requests
+* custom-ignoring spam application requests
```

Comment:
* Retry all requests that receive a 429 response
* Queue application requests in case you receive a temporary increase in applications that exceeds Lever's application POST rate limit
* Implement spam mitigations such as captchas and session-based rate limits

If you do not retry application POST requests upon receiving a 429 response, you will unfortunately lose candidate applications.